src/app/app.component.ts

```diff
@@ -67,16 +87,22 @@ export class AppComponent implements OnInit, OnDestroy {
     - Subscribe to changes
     - Check the received value using the areAllValuesTrue function and pass them to the isLoading variable. */
     // YOUR CODE STARTS HERE
+    combineLatest(this.mockDataService.getCharactersLoader(), this.mockDataService.getPlanetLoader()).subscribe(
+      ([charactersLoader, planetsLoader]) => {
+        this.isLoading = this.areAllValuesTrue([charactersLoader, planetsLoader]);
+      }
+    );
```

Comment:
Here is where you should've saved the subscription for unsubscribing from it in the ngOnDestroy method. Now it's a memory leak.